docs/mkdocs.yml

```diff
@@ -87,6 +87,7 @@ nav:
       - run:
         - run get: 'cmd/flow/run/run-get.md'
         - run list: 'cmd/flow/run/run-list.md'
+        - run cancel: 'cmd/flow/run/run-cancel.md'
```

Comment:
Let's sort commands alphabetically in the TOC so that it's easier for users to find them

Reply:
Will pay attention to this in the future. 😄